### DISCUSSION TITLE
Please update timeline for alternate/opposite sides  also

### DISCUSSION THREAD

**Discussion by harshlibra08** [mantinedev/mantine#5325] [Feature requests]
### What package has an issue?

@mantine/core

### Describe the bug

The timeline section only contains content on one side , kindly update it so that it appears both the sides horizontally and vertically

### What version of @mantine/* packages do you have in package.json? (Note that all @mantine/* packages must have the same version in order to work correctly)

7.1.0

### If possible, please include a link to a codesandbox with a minimal reproduction

https://codepen.io/code-boxx/pen/BamPEqz

### Do you know how to fix the issue

None

### Do you want to send a pull request with a fix?

None

**Comment by UsmanS2** [comment]
I also want this :)

**Comment by matronator** [comment]
Please add this.

**Comment by reeteshsudhakar** [comment]
would also love to see this! bumping for visibility :)

**Comment by TijlDeclerckWd** [comment]
Yes, also want to see this!

**Comment by VukiMusician** [comment]
Also would want this!

**Comment by sovetski** [comment]
+++++1 for this

@harshlibra08 your codepen link is expired, I guess it was something like

![image](https://github.com/user-attachments/assets/7f210ca1-b4ce-4fcb-8b1f-e094910056c6)

**Reply by timtucker-dte** [reply]
This would be great -- react-vertical-timeline-component is one of the few options I know of that supports this type of view:
https://www.npmjs.com/package/react-vertical-timeline-component

The visual difference makes a lot of sense when you have events that fall into 2 distinct categories.

Some example use cases:
- Income vs. expenses
- Requests for work vs. work completed
- Orders vs. shipments
- Home games vs. away games

**Comment by adhamaa** [comment]
please add this feature!

**Comment by Ri1a** [comment]
This would be a really great feature!

**Comment by AdrienHolvoet** [comment]
Up on this too
